docs/known_issues.rst

```diff
@@ -75,6 +75,39 @@ Both will throw an exception if units do not cancel, e.g.::
 
 See: https://github.com/astropy/astropy/issues/7582
 
+Quantities Lose Their Units with Operations on pandas.Series
+------------------------------------------------------------
+
+Quantities may work with certain operations on `~pandas.Series` but
+this behaviour is not tested. Quantities are subclassed from ``numpy``'s `~numpy.ndarray`
+and they do not behave same as `~pandas.Series` objects.
+For example one know issue is that multiplying a `~pandas.Series` object
+with a unit will *not* return a ``Quantity`` object. It will return a `~pandas.Series`
+object without any unit::
+
+   >>> import pandas as pd # doctest: +SKIP
+   >>> import astropy.units as u # doctest: +SKIP
+   >>> a = pd.Series([1., 2., 3.]) # doctest: +SKIP
+   >>> a * u.m # doctest: +SKIP
```

Comment:
Do run the tests is possible. If the test fails with `pandas` 1.4 then use
```suggestion
.. doctest-requires:: pandas>=1.5

   >>> import pandas as pd # doctest: +SKIP
   >>> import astropy.units as u # doctest: +SKIP
   >>> a = pd.Series([1., 2., 3.]) # doctest: +SKIP
   >>> a * u.m # doctest: +SKIP
```
Also, if different versions of `pandas` produce different outcomes then it's important to document that, so we might need multiple sets of tests.

Reply:
Pandas 1.4 should be dropped according to [SPEC 0](https://scientific-python.org/specs/spec-0000/), I'm not sure if we should write specific docs for that? `olddeps` should probably drop pandas 1.4 soon?

Reply:
Dropping support for `pandas` 1.4 sounds like a good idea, but the pull request doing that should be merged first.

Reply:
Sounds good, created https://github.com/astropy/astropy/pull/16023